modules/datastore/src/Controller/QueryDownloadController.php

```diff
@@ -131,4 +133,56 @@ private function sendRow($handle, array $row) {
     flush();
   }
 
+  /**
+   * Set up the Stream query result as json objects.
+   *
+   * @param \Drupal\datastore\Service\DatastoreQuery $datastoreQuery
+   *   A datastore Query object.
+   * @param \RootedData\RootedJsonData $result
+   *   Query result.
+   */
+  protected function loadJson(DatastoreQuery $datastoreQuery, RootedJsonData $result) {
```

Comment:
Can we move this below the streamJsonResponse() method, to keep the order more predictable?